### ISSUE TITLE
Dish IH tokens when we dish IH

### ISSUE THREAD

**Issue by tamarandom** [TECommons/coordination#380]
## 📋 Anything to add?

(please reword this to make it more clear) ;)

## 🎉 Subtasks

- [ ] 
- [ ] 
- [ ] 

## 🤼‍ Reviewer

@

**Comment by freedumbs00** [comment]
tokens minted and dispersed

**Comment by freedumbs00** [comment]
Tokens minted and dispersed

**Comment by freedumbs00** [comment]
Tokens minted and dispersed for the last two quants.

**Comment by freedumbs00** [comment]
Tokens minted and dispersed

**Comment by freedumbs00** [comment]
Tokens Minted and dished

**Comment by tamarandom** [comment]
Hey @freedumbs00 - was this not done this Sprint!?

**Comment by cranders1** [comment]
on it
